nodeJS/getting-started/Introduction.md

```diff
@@ -68,8 +68,9 @@ While you may have learned React (or any other frontend framework) before, eithe
 
 1. [This short module](https://developer.mozilla.org/en-US/docs/Learn/Server-side/First_steps) on "The Server Side" from MDN is a great source for the background knowledge you need. Read through at least the first two articles posted under the 'Guides' section: Introduction to the server side and Client-Server Overview. The other two are interesting and worth reviewing, but less relevant to our immediate concerns.
 2. To gain a little more insight into the nature of Node, and to unpack the rest of the above definition, read [this article](https://medium.freecodecamp.org/what-exactly-is-node-js-ae36e97449f5). There's a long, but *really* fantastic video linked in that article, don't skip it!
```

Comment:
If it's no longer linked here, it may be good to remove the mention of it here then so people don't get confused.

Reply:
Thanks for telling me about the checkboxes!  Yeah... I'm new to Markdown! :P 

And ugggh sorry about not removing the previous mention, I did originally but since I did it on the wrong branch I had to redo everything and I forgot the second time.  

I'm going to try to correct everything including the whitespace.

Reply:
I'm assuming that by the extra whitespace you were referring to me using two spaces after a period instead of one?  Is this considered bad practice?

Reply:
Meaning after the number 3, you had extra space before the text compared to the other numbers in the list. It just didn’t match.